specification/network/resource-manager/Microsoft.Network/stable/2021-03-01/bastionHost.json

```diff
@@ -588,6 +588,19 @@
           "type": "string",
           "description": "FQDN for the endpoint on which bastion host is accessible."
         },
+        "sku": {
```

Comment:
The sku should be outside properties envelope.
Refer RPC documentation: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#put-resource